q_yaar_platform/treasure_hunt/popo/time_interval.py

```diff
@@ -0,0 +1,32 @@
+from dataclasses import dataclass
+
+from common.popo import PopoBase
+
+
+@dataclass(init=True, repr=True, eq=False)
+class TimeIntervalConfig(PopoBase):
+    _from: int  # epoch timestamp
+    _to: int  # epoch timestamp
+
+    @classmethod
+    def from_json(cls, config: dict) -> "TimeIntervalConfig":
+        if not config:
+            return cls.default()
+        return cls(_from=config.get("from"), _to=config.get("to"))
+
+    def to_json(self) -> dict:
+        return {"from": self._from, "to": self._to}
+
+
+@dataclass(init=True, repr=True, eq=False)
+class TimeIntervalListConfig(PopoBase):
```

Comment:
Frontend will need to compute and send this structure during game creation.
[{"from": epoch1, "to": epoch2}, {"from": epoch3, "to": epoch4},...]
User will select date and time from a calendar or something. Frontend will fetch this epoch and send the json.
@paragjnath